### ISSUE TITLE
Question about voting.

### ISSUE THREAD

**Issue by LMD0311** [zyh16143998882/IDPT#3]
Thanks for your inspiring work!

Could you please tell me if it is true that simply using IDPT can only achieve $93.3$ on ModelNet, then with voting mechanism can achieve $94.4$? It does not seem to be mentioned in your paper.

**Comment by zyh16143998882** [comment]
Thank you for your interest, you raise a good question.
Firstly, following previous works (Point-Bert, Point-MAE, ACT, etc.), we report the results after voting, as mentioned in the paper section 4.2.2. 

We have also tried several different random seeds as in previous work and reported the best of them with a vote result of 94.4. In fact, with IDPT, we can achieve many results above 93.3%, for example, we once obtained a non-voting accuracy of 93.7%. However, since the voting using this checkpoint did not exceed 94.4%, we did not report this result in our paper. You can train our code yourself using different seeds for validation.

**Comment by LMD0311** [comment]
Thanks for your patient reply! @zyh16143998882 

I have reproduced your results base on your code (all random seeds are set to be $0$), here are my results on ModelNet 40:

| Methods          | results in paper       | Reproduce w/o voting | w/ voting | Tunable parameters (including head) |
| ---------------- | ---------------------- | ------------------- | --------- | ---------------------------------- |
| Only Head Tuning | 93.2                   | 92.34               | 92.75     | 1.24%                              |
| Full Fine-tuning | 93.8                   | 92.74               | 93.27     | 100%                               |
| IDPT             | 94.4 (93.3 w/o voting) | 93.07               | 93.68     | 7.24%                              |

The validity of your proposed method can be seen from the results above, and as a comparison, I would like to know whether all your methods eval on ModelNet 40 in paper use voting mechanism (including full fine-tuning and fix encoder)?  Are all methods mentioned in Table 3 using voting mechanism?

**Comment by zyh16143998882** [comment]
Thank you for the validation.
1) For a fair comparison, we follow the practice of previous papers (Point-BERT, Point-MAE, ACT, etc.), and all ModelNet40 results in the paper use the voting mechanism. All ScanObjectNN results do not use the voting mechanism.
2) For a fair comparison, the results of the other methods in our Table 3 were obtained from previous papers (e.g., Point-BERT, Point-MAE, ACT, etc.). As for whether they all use the voting mechanism, as far as I know, all the "with Self-Supervised Representation Learning" methods use the voting mechanism. Point-MLP also uses voting in "Supervised Learning Only".

**Comment by LMD0311** [comment]
> Thank you for the validation.
> 
> 1. For a fair comparison, we follow the practice of previous papers (Point-BERT, Point-MAE, ACT, etc.), and all ModelNet40 results in the paper use the voting mechanism. All ScanObjectNN results do not use the voting mechanism.
> 2. For a fair comparison, the results of the other methods in our Table 3 were obtained from previous papers (e.g., Point-BERT, Point-MAE, ACT, etc.). As for whether they all use the voting mechanism, as far as I know, all the "with Self-Supervised Representation Learning" methods use the voting mechanism. Point-MLP also uses voting in "Supervised Learning Only".

Many thanks! 

By the way, I find a unused class named '''PointTransformerAdapter'''. Have you ever tried Adapter-like methods in your research? Any empirical results？
https://github.com/zyh16143998882/ICCV23-IDPT/blob/f6c2c0ee05a7e7e6fcd37ab7c0c20392360890a6/models/Point_MAE.py#L686

**Comment by zyh16143998882** [comment]
Yes, we have tried Adapter before and found that it still has a big gap with fine-tuning. Meanwhile, we mainly explored prompt, so we didn't mainly introduce Adapter.

**Comment by LMD0311** [comment]
Thank you again for your patience! I will close the issue.
